robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAssetManager.java.vm

```diff
@@ -271,19 +268,21 @@ public final class ShadowAssetManager {
 
   @HiddenApi @Implementation
   public static void applyThemeStyle($ptrClass theme, int styleRes, boolean force) {
-    if (!APPLIED_STYLES.containsKey(theme)) {
-      APPLIED_STYLES.put(theme, new LinkedList<OverlayedStyle>());
-    }
-
     ResourceLoader resourceLoader = getInstance().getResourceLoader();
     ShadowAssetManager assetManager = shadowOf(getInstance().getAssets());
 
+	final Map<$ptrClassBoxed, List<OverlayedStyle>> appliedStyles = assetManager.appliedStyles;
```

Comment:
Thanks @jongerrish, my apologies. Another minor disadvantage of having velocity source files - Eclipse treats it as plain text instead of Java source and uses different formatting rules.